metadata-ingestion/src/datahub/ingestion/source/tableau/tableau.py

```diff
@@ -2667,9 +2679,10 @@ def update_datasource_for_field_upstream(
         # Fetch field upstreams and arrange them in map
         field_vs_upstream: Dict[str, dict] = {}
         for field_upstream in self.get_connection_objects(
-            field_upstream_query,
-            c.FIELDS_CONNECTION,
-            {c.ID_WITH_IN: field_ids},
+            query=field_upstream_query,
+            connection_type=c.FIELDS_CONNECTION,
+            query_filter={c.ID_WITH_IN: field_ids},
+            page_size_override=page_size_override,
```

Comment:
not sure about the need to pass page_size_override arg here, given that it can directly access the config. Also no harm using same config for published datasource, right ?

Reply:
Proposal PR for fine-grained page sizes PR #12354